diff --git a/cmake/OCL.cmake b/cmake/OCL.cmake
index 773c6f6..c334a1b 100644
--- a/cmake/OCL.cmake
+++ b/cmake/OCL.cmake
@@ -42,11 +42,12 @@ if (NOT TEST_CODE_OBJECT_VERSION_NONE_RESULT)
   set(CLANG_OCL_FLAGS ${CLANG_OCL_FLAGS} -Xclang -mcode-object-version=none)
 endif()
 
+include(GNUInstallDirs)
 set (BC_EXT .bc)
 set (LIB_SUFFIX ".lib${BC_EXT}")
 set (STRIP_SUFFIX ".strip${BC_EXT}")
 set (FINAL_SUFFIX "${BC_EXT}")
-set (INSTALL_ROOT_SUFFIX "amdgcn/bitcode")
+set (INSTALL_ROOT_SUFFIX "${CMAKE_INSTALL_DATADIR}/amdgcn/bitcode")
 
 # Set `inc_options` to contain Clang command-line for include directories for
 # current source directory.
diff --git a/cmake/Packages.cmake b/cmake/Packages.cmake
index 07c60eb..bc309ec 100644
--- a/cmake/Packages.cmake
+++ b/cmake/Packages.cmake
@@ -16,7 +16,7 @@ configure_file(AMDDeviceLibsConfig.cmake.in
   @ONLY)
 
 
-set(install_path_suffix "amdgcn/bitcode")
+set(install_path_suffix "${CMAKE_INSTALL_DATADIR}/amdgcn/bitcode")
 
 # Generate the install-tree package.
 # We do not know the absolute path to the intall tree until we are installed,
